.github/workflows/compaund-java-multi-project-build.yml

```diff
@@ -0,0 +1,69 @@
+name: Building module in multi-project
+
+on:
+  workflow_call:
+    inputs:
+      runsOn:
+        required: false
+        type: string
+        default: 'ubuntu-20.04'
+      versionSuffix:
+        required: false
+        type: string
+        default: ''
+        description: "version suffix for module's final version"
```

Comment:
Could we calculate the version here instead of passing as variable

Reply:
Yes, we can. But it will be done on each workflow call.
Even if we move the calculation inside the workflow we still will have to pass a parameter to indicate whether a release or snapshot should be published.

I can change the workflow and add `release` parameter with boolean type instead

Reply:
Yes, please